### ISSUE TITLE
_field_caps API fails with cross cluster search

### ISSUE THREAD

**Issue by jimmyjones2** [opendistro-for-elasticsearch/index-management#424]
Reported in a few forum threads:
* https://discuss.opendistrocommunity.dev/t/cross-cluster-search-with-secured-clusters/5162/7
* https://discuss.opendistrocommunity.dev/t/cross-cluster-search-with-secured-local-and-remote-clusters/5109/3
* https://discuss.opendistrocommunity.dev/t/kibana-ccs-fails/5405

Was working on 1.9.0, 1.10.1, 1.11. Fails on 1.12.0, 1.13.0 and 1.13.1

I’m trying to use Kibana to search indices in a remote cluster - following [CCS docs](https://opendistro.github.io/for-elasticsearch-docs/docs/security/access-control/cross-cluster-search/)

When I try and add an index pattern for odfe-cluster1:books I can see a 404 to http://127.0.0.1:5601/api/index_patterns/_fields_for_wildcard?pattern=odfe-cluster1%3Abooks&meta_fields=_source&meta_fields=_id&meta_fields=_type&meta_fields=_index&meta_fields=_score

Digging into the source for what that calls, it looks like the `_field_caps` API doesn’t support CCS:
```
curl -XGET -k -u booksuser:password 'https://localhost:9250/odfe-cluster1:books/_field_caps?fields=_id'
{"error":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index [odfe-cluster1:books]","resource.type":"index_or_alias","resource.id":"odfe-cluster1:books","index_uuid":"_na_","index":"odfe-cluster1:books"}],"type":"index_not_found_exception","reason":"no such index [odfe-cluster1:books]","resource.type":"index_or_alias","resource.id":"odfe-cluster1:books","index_uuid":"_na_","index":"odfe-cluster1:books"},"status":404}
```

As this works without the security plugin installed, I presume the issue is there.

**Comment by thalurur** [comment]
The fix is released as part of 1.13.2 - https://github.com/opendistro-for-elasticsearch/index-management/releases/tag/v1.13.2.0

**Comment by jimmyjones2** [comment]
Thanks @thalurur
